EvidenceApi/StaffSelectedDocumentTypes.json

```diff
@@ -164,14 +164,19 @@
         "title": "EU identity card",
         "description": "Photo page"
       },
+      {
+        "id": "resident-permit",
```

Comment:
For reference, this was removed in the spreadsheet. To confirm, we can add new fields but not remove them, so we'll keep resident-permit